spharpy/spherical.py

```diff
@@ -767,3 +768,404 @@ def sph_identity_matrix(n_max, type='n-nm'):
+    - $N_{nm}$ is the normalization term
+    - $T_{nm}$ is a term that depends on whether the
+     harmonics are real or complex
+    - $\theta$ is the colatitude (angle from the positive z-axis)
+    - $\phi$ is the azimuth (angle from the positive x-axis in the xy-plane)
+
+    The normalization term Nnm is given by:
+
+    .. math::
+        N_{nm}^{\text{SN3D}} =
+        \sqrt{\frac{2n+1}{4\pi} \frac{(n-|m|)!}{(n+|m|)!}}
+
+        N_{nm}^{\text{N3D}} = N_{nm}^{\text{SN3D}} \sqrt{\frac{2n+1}{2}}
+
+        N_{nm}^{\text{MaxN}} = ... (max of N3D)
+
+    The associated Legendre function Pnm is given by:
+
+    .. math::
+        P_{nm}(x) = (1-x^2)^{|m|/2} (d/dx)^n (x^2-1)^n
+
+    The term Tnm is given by:
+        - For complex-valued harmonics:
+            .. math::
+                T_{nm} = e^{im\phi}
+        - For real-valued harmonics:
+            .. math::
+                T_{nm} = \begin{cases}
+                            \cos(m\phi) & \text{if } m \ge 0 \\
+                            \sin(m\phi) & \text{if } m < 0
+                        \end{cases}
+
+    The spherical harmonics are orthogonal on the unit sphere, i.e.,
+
+    .. math::
+        \int_{sphere} Y_{nm} Y_{n'm'}* d\omega = \delta_{nn'} \delta_{mm'}
+
+    where:
+    - $*$ denotes the complex conjugate
+    - $\delta$ is the Kronecker delta
+    - $d\omega$ is the differential solid angle
+    - The integral is over the entire sphere
+
+    The class supports the following conventions:
+
+    - basis_type: Defines the type of spherical harmonic basis.
+    It can be either 'complex' or 'real'.
+        - ``'complex'``: Uses complex-valued spherical harmonics.
+        - ``'real'``: Uses real-valued spherical harmonics.
+
+    - normalization: Defines the normalization convention.
+     It can be 'n3d', 'maxN', or 'sn3d'.
+        - ``'n3d'``: Uses the 3D normalization
+        (also known as Schmidt semi-normalized).
+        - ``'maxN'``: Uses the maximum norm
+        (also known as fully normalized).
+        - ``'sn3d'``: Uses the SN3D normalization
+        (also known as Schmidt normalized).
+
+    - channel_convention: Defines the channel ordering convention.
+    It can be either 'acn' or 'fuma'.
+        - ``'acn'``: Follows the Ambisonic Channel Number (ACN) convention.
+        - ``'fuma'``: Follows the Furse-Malham (FuMa) convention.
+
+    - inverse_transform: Defines the type of inverse transform.
+     It can be 'pseudo_inverse', 'quadrature', or None.
+        - ``'pseudo_inverse'``: Uses the Moore-Penrose pseudo-inverse
+         for the inverse transform.
+        - ``'quadrature'``: Uses quadrature for the inverse transform.
+        - ``None``: No inverse transform is applied.
+
+    Parameters
+    ----------
+    n_max : int
+        Maximum spherical harmonic order
+    coordinates : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>` or
+        `sp.SamplingSphere <spharpy:classes/spharpy.samplings.coordinates>`
```

Comment:
```suggestion
    coordinates : :py:class:`~pyfar.classes.coordinates.Coordinates ` or :py:class:`~spharpy.samplings.coordinates.SamplingSphere`
```
the ~ makes it in short form in the doc.